### ISSUE TITLE
Start Building Sentiment Dataset for AI Assignment

### ISSUE THREAD

**Issue by radioheadfan123** [radioheadfan123/ffxiv-log-analyzer#10]
This issue tracks the initial steps for the AI class assignment: building a sentiment dataset for later fine-tuning.

**Assignment Overview:**
<img alt="Assignment instructions screenshot" width="1238" src="https://github.com/user-attachments/assets/b92a42c5-e3f0-42c6-b7c0-797b2aa55f6d" />

Key requirements:
- Create a CSV file with columns: split (train/validation/test), id, text, label (negative/neutral/positive), source, language.
- Data split: 80% train, 10% validation, 10% test. Each label (negative/neutral/positive) should represent 25–45% in the training set.
- Size guidance: 150–300 rows of data. More rows are acceptable if quality is high.
- Data sources: Choose one theme (e.g., public reviews, course feedback, news/forum comments).
- Avoid any personally identifiable information (PII) or content from minors. Honor Terms of Service for data sources.
- Follow labeling rules strictly:
  - Positive: clear satisfaction/praise
  - Negative: clear dissatisfaction/complaint
  - Neutral: mixed/factual/unclear
  - Prefer intent over words (e.g., sarcasm marked neutral if unclear)

**Deliverable:**
- CSV file named as specified: `sentiment-data-{your_last_name}.csv`

---

**Tasks:**
- [ ] Decide on a theme for data sources
- [ ] Gather text samples and label them according to rules
- [ ] Create CSV with required columns
- [ ] Ensure balanced splits and label distribution
- [ ] Check for PII and compliance with data rules
- [ ] Upload CSV deliverable

Refer to the attached image for full assignment instructions.
